charts/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -236,17 +236,17 @@ spec:
           {{- end }}
           - name: DD_REMOTE_CONFIGURATION_ENABLED
             value: {{ include "clusterAgent-remoteConfiguration-enabled" . | quote }}
-                    {{- if .Values.datadog.apm.instrumentation.enabled }}
+          {{- if .Values.datadog.apm.instrumentation.enabled }}
           - name: DD_APM_INSTRUMENTATION_ENABLED
             value: "true"
           {{- end }}
           {{- if .Values.datadog.apm.instrumentation.enabledNamespaces }}
           - name: DD_APM_INSTRUMENTATION_ENABLED_NAMESPACES
             value: {{ .Values.datadog.apm.instrumentation.enabledNamespaces | toJson | quote }}
           {{- end }}
-          {{- if .Values.datadog.apm.instrumentation.disabledNamespaces }}
+          {{- if ne (include "apmInstrumentation.disabledNamespaces" .) "" }}
```

Comment:
Should we default `apmInstrumentation.disabledNamespaces` to an empty list when not defined to simplify this check?

Reply:
`apmInstrumentation.disabledNamespaces` is a string; it's "" by default.

https://github.com/DataDog/helm-charts/blob/main/charts/datadog/templates/_helpers.tpl#L618-L631 has similar pattern and usage https://github.com/DataDog/helm-charts/blob/main/charts/datadog/templates/_containers-common-env.yaml#L27-L30